src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -132,7 +132,7 @@ public static FormulaValue Concatenate(IRContext irContext, StringValue[] args)
         // Convert string to number
         public static FormulaValue Value(EvalVisitor runner, EvalVisitorContext context, IRContext irContext, FormulaValue[] args)
         {
-            return Value(CreateFormattingInfo(runner), irContext, args);
+            return Value(FormattingInfoHelper.FromEvalVisitor(runner), irContext, args);

```

Comment:
>FormattingInfoHelper.FromEvalVisitor(runner) [](http://example.com/codeflow?start=25&length=44)

can we make this an extension method, so that we can then just say:

runner.GetFormattingInfo()

Reply:
updated.